### PR TITLE
Start Feature/add rossella

### PR DESCRIPTION
**Pull request by rossellamascia**
This is a draft of my project

### CODE REVIEW

File: authors.json [hunk 1]
```diff
@@ -25,6 +25,10 @@
     "country": "Italy"
   },
   {
+    "name": "Rossella",
+    "githubNick": "_RedRoss_",
+    "country": "Italy"
+ },
```
Comment (RickyDoesCode) [inline authors.json:31]: Try adding another opening curly bracket for the person after you.

Comment (RickyDoesCode) [reply authors.json:31]: @rossellamascia

**Comment by pixari** [comment]
Hi @rossellamascia!

There are some things to fix! :)

1) You don't have to put on the repo every file you have in the project. You can exclude the cache files and the npm modules, for example. The other developers (or the future you) will download them again, there's no need to store them on git. Please, exclude all the useless files from git using .gitignore.

2) Nice design, but doesn't fulfill the requirements. In the sketch there aren't work links, timeline, skills, etc. etc.
It's enough to satisfy the acceptance criteria.

3) What is the "dist" folder? And why you need the package.json?

**Comment by RickyDoesCode** [comment]
Hi @rossellamascia, since you've left this PR open, and made no more changes to it as requested, we'll close this PR for now. Feel free to reopen if you want to resubmit your task.

**Pull request closed without merging**